README.md

```diff
@@ -66,6 +66,15 @@ To install this expansion, please follow these steps:
 - If you already have a previous version of the devkit, update the pip requirements (see [details](https://github.com/nutonomy/nuscenes-devkit/blob/master/setup/installation.md)): `pip install -r setup/requirements.txt`
 - Get started with the [tutorial](https://github.com/nutonomy/nuscenes-devkit/blob/master/python-sdk/nuscenes/python-sdk/tutorials/map_expansion_tutorial.ipynb).
 
+## nuScenes-lidarseg expansion
+In June 2020 we published a nusScenes-lidarseg expansion which contains semantic labels of various pointclouds.
```

Comment:
nuScenes
various? Explain that these are the pointclouds of the 40k keyframes in nuScenes.